pkg/openshift/templates/push-ups.json.tpl.go

```diff
@@ -0,0 +1,146 @@
+package templates
+
+//PushUPSTemplate defines the template for deploying a push via UPS environment service to openshift 3
+var PushUPSTemplate = `
+{{define "push-ups"}}
+{
+  	"kind": "Template",
+  	"apiVersion": "v1",
+  	"metadata": {
+    	"name": "RHMAP UPS",
+    	"annotations": {
+      		"description": "RHMAP ups",
+			"dependencies": "data-mysql",
```

Comment:
at the moment I'm using space-seperated in that work, would you prefer comma-seperated? I don't really have a strong opinion either way.